moderators.md

```diff
@@ -0,0 +1,137 @@
+
+## Topics
+
+We encourage moderators to contribute [R-multiverse topics](https://r-multiverse.org/topics/) about their fields of expertise.
+To do so, submit a pull request to <https://github.com/r-multiverse/topics> to add a text file describing the subject matter ([example here](https://github.com/r-multiverse/topics/blob/main/bayesian)).^[Through triage access, R-multiverse [moderators](governance.md#moderator) are authorized to merge these pull requests.
+Please only edit the plain files without file extensions.
+All the HTML files are automatically generated.]
+A bot periodically rebuilds the website based on these text files.
+Seasoned [contributors](governance.md#contributor) will see your topic [listed here](https://r-multiverse.org/topics/) and add keywords to their [contributions](https://github.com/r-multiverse/contributions/pulls) accordingly.
+
+## Discussions
+
+[Moderators](governance.md#moderator) help answer questions on the [`r-multiverse/help`](https://github.com/r-multiverse/help) [issue](https://github.com/r-multiverse/help/issues) and [discussion](https://github.com/r-multiverse/help/discussions) forums.
+Questions may ask about specific packages, package reviews, the workings of R-multiverse, etc.
+A [moderator](governance.md#moderator) may close an [issue](https://github.com/r-multiverse/help/issues) or [discussion](https://github.com/r-multiverse/help/discussions) thread if it originally came from a [contributor](governance.md#contributor) or [user](governance.md#user) and the original issue is resolved.
+[Moderators](governance.md#moderator) should not close issues or discussions created by [administrators](team.md#administrators), including [official proposals](governance.md#proposals) to modify [R-multiverse policies](policies.md).
+
+## Enforcement
+
+[Moderators](governance.md#moderator) must understand [all R-multiverse policies](policies.md) and be prepared to enforce them.
+[Moderators](governance.md#moderator) must respond immediately and quickly to the following rare cases.
+Each incident, the actions taken to respond to the incident, and the resolution must be reported as an issue at <https://github.com/r-multiverse/help/issues> with the label "policy violation".
+If you do not feel comfortable taking direct action yourself, contact an [administrator](team.md#administrators) immediately and ask the [administrator](team.md#administrators) handle the situation.
+
+1. In case of an extreme policy violation, or repeated violations of the [code of conduct](conduct.md),
+  block the user from [R-multiverse GitHub repositories](https://github.com/r-multiverse)
+  by adding the GitHub username to the [list of blocked users](https://github.com/organizations/r-multiverse/settings/blocked_users).
+1. If an R-multiverse package or [R-multiverse stakeholder](governance.md#r-multiverse-stakeholders) violates any applicable laws, contact the appropriate law enforcement agency.
```

Comment:
What country laws are in play here, and how can a moderator know who to call?

Reply:
12532f0b0cf7f3f7526c0592865a6d1167e51284 reverts to a previous version of this point: "ask an administrator to contact the appropriate law enforcement agency".

The [Acceptable Use Policy](https://r-multiverse.org/aup.html#prohibited-content) is deliberately general in scope:

> Users and Contributors must not submit, post, or share any content that:
> 
> * Illegal or Unlawful: Violates any applicable laws, including content that promotes illegal activities, constitutes fraud, or violates intellectual property rights or other laws, regulations, and guidelines in your jurisdiction.

The "any applicable laws" phrase is meant to capture whatever applies to the specific case at hand. A complicated situation may require a conversation among admins.